src/main/protobuf/sorting.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package cs332.distributedsorting;
+
+service Sorting {
+  rpc Handshake (HandshakeRequest) returns (HandshakeResponse) {}
+  rpc SendData(SendDataRequest)returns (SendDataResponse){}
+}
+
+
+message SendDataRequest{
+  string data = 1;
```

Comment:
protobuf supports `bytes` type. How about using `bytes` instead of `string`?
https://developers.google.com/protocol-buffers/docs/proto3#scalar